example/src/child-component/index.js

```diff
@@ -7,7 +7,7 @@ import computed from "./computed";
 
 export const addState = withState({ effects: effectDefs, initialState, computed });
 
-export const ChildComponent = (props, { state, effects }) => (
+export const ChildComponent = ({ state, effects }) => (
```

Comment:
In a simple todo-like app, this might be a nice to have.  But I think as application complexity grows, it would become the bane of my existence :P  We could provide a helper function that spreads `state` and `effects` - essentially an opt-in.